server/index.js

```diff
@@ -1,62 +1,25 @@
 const express = require('express');
 const path = require('path');
 const bodyParser = require('body-parser');
+
 const app = express();
-const menus = require('../database/database');
+const { getMenu } = require('./controllers');
+const { createMenu } = require('./controllers');
+const { updateMenu } = require('./controllers');
+const { deleteMenu } = require('./controllers');
```

Comment:
You can condense line 6-9 into one line:
const { getMenu, createMenu, updateMenu, deleteMenu }